pkg/export/setup/setup.go

```diff
@@ -116,8 +123,8 @@ func FromFlags(a *kingpin.Application, userAgent string) func(log.Logger, promet
 	a.Flag("export.label.project-id", fmt.Sprintf("Default project ID set for all exported data. Prefer setting the external label %q in the Prometheus configuration if not using the auto-discovered default.", export.KeyProjectID)).
 		Default(opts.ProjectID).StringVar(&opts.ProjectID)
 
-	a.Flag("export.user-agent", "Override for the user agent used for requests against the GCM API.").
-		Default(userAgent).StringVar(&opts.UserAgent)
+	a.Flag("export.user-agent-mode", "Mode for user agent used for requests against the GCM API.").
```

Comment:
Nit - please mention the valid value in the helper text like the following one:

https://github.com/GoogleCloudPlatform/prometheus-engine/blob/d6f6391e42c24ea252c632c35b3fe587bf327bf7/pkg/export/setup/setup.go#L159

Reply:
Done.